scripts/release.js

```diff
@@ -167,18 +183,63 @@ if (!fs.existsSync('dependency-check-app')){
 process.chdir(path.join(__dirname, '../dependency-check-app'));
 spawnOrFail('npm', ['init -y']);
 spawnOrFail('npm', ['install react react-dom']);
-spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion} styled-components styled-system`]);
-spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
-checkWarning('npm', ['install -q'], null, 'amazon-chime-sdk-component-library-react');
+spawnOrFail('npm', [
+  `install amazon-chime-sdk-js@${updatedSdkVersion} styled-components styled-system`,
+]);
+spawnOrFail('npm', [
+  `install ../amazon-chime-sdk-component-library-react-${versionString}.tgz`,
+]);
+checkWarning(
```

Comment:
You already install react right above right? What does this do then?

Reply:
This is to install the tar file to a newly created folder `../dependency-check-app`,